src/com/facebook/buck/jvm/kotlin/KotlinConfiguredCompilerFactory.java

```diff
@@ -65,6 +65,7 @@ public CompileToJarStepFactory configure(
         kotlinBuckConfig.getKotlinHomeLibraries(),
         kotlinArgs.getExtraKotlincArguments(),
         kotlinArgs.getAnnotationProcessingTool().orElse(AnnotationProcessingTool.KAPT),
+        kotlinArgs.getKaptApOptions(),
```

Comment:
would be good to support macros as well in https://github.com/facebook/buck/commit/3e0b4cc444d86f565124807f729bbe7d9748c83c#diff-8bd16d403f308d9df417653053234adb in kaptApOptions